docs/snippets/technical-reference/tasks/custom_task_selfinstruct_haikus_docs.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
```

Comment:
why the new file and not the old one?

Reply:
It was just a name change, I can revert it, but thought it would be useful to state that this snippet is for the docs